locales/ar/common.json

```diff
@@ -247,6 +247,7 @@
     "shared": "تم نسخ الرابط إلى الحافظة!",
     "sidebar": {
         "search-juz": "ابحث عن جزء",
+        "search-hizb": "ابحث عن حزب",
```

Comment:
@ahmedriad1 usually we add a new string in the English file only and use [Lokalise](https://lokalise.com/) to generate the same string in all the other languages we support. I will do that in a follow up. We should undo `/ar/quran-reader.json` and `/ar/common.json`.

Reply:
I see. I'll remove the arabic translation now